readme.md

```diff
@@ -38,6 +45,29 @@ The rules will only activate in test files.
 - [prefer-power-assert](docs/rules/prefer-power-assert.md) - Allow only use of the asserts that have no [power-assert](https://github.com/power-assert-js/power-assert) alternative.
 
 
+## Recommended configuration
+
+This plugin exports a `recommended` configuration that enforces good practices.
```

Comment:
Maybe linkify ``recommended` configuration` to the relative file path of index.js so users can easily see the recommended config.